sentry-android-core/src/main/java/io/sentry/android/core/internal/util/ScreenshotUtils.java

```diff
@@ -23,20 +20,16 @@ public class ScreenshotUtils {
   private static final long CAPTURE_TIMEOUT_MS = 1000;
 
   public static @Nullable byte[] takeScreenshot(
-      final @NotNull Activity activity,
-      final @NotNull ILogger logger,
-      final @NotNull BuildInfoProvider buildInfoProvider) {
-    return takeScreenshot(
-        activity, AndroidMainThreadChecker.getInstance(), logger, buildInfoProvider);
+      final @NotNull Activity activity, final @NotNull ILogger logger) {
```

Comment:
This is being used by RN, we either keep this signature of provide a fix on their repo.

Reply:
I reverted it, so that if we are going to need it in the future we don't have to change RN again